content/integrations/cloud_foundry.md

```diff
@@ -0,0 +1,283 @@
+
+### Upload Datadog's release to your BOSH Director
+
+~~~
+# BOSH CLI v1
+bosh upload release https://github.com/DataDog/datadog-firehose-nozzle-release/releases/download/61/datadog-firehose-nozzle-release-61.tgz
+
+# BOSH CLI v2
+bosh upload-release https://github.com/DataDog/datadog-firehose-nozzle-release/releases/download/61/datadog-firehose-nozzle-release-61.tgz
+~~~
+
+If you'd like to create your own release, see the [Datadog Firehose Nozzle release repository](https://github.com/DataDog/datadog-firehose-nozzle-release).
+
+### Configure a UAA client
+
+In the manifest that contains your UAA configuration, add a new client for the Datadog Nozzle so the job(s) can access the Firehose:
+
+~~~
+uaa:
+  clients:
+    datadog-firehose-nozzle:
+      access-token-validity: 1209600
+      authorities: doppler.firehose
+      authorized-grant-types: client_credentials
+      override: true
+      scope: doppler.firehose
+      secret: <MAKE_IT_A_GOOD_ONE>
+~~~
+
+And redeploy the deployment to add the user.
+
+### Add Nozzle jobs
+
+Configure one or more Nozzle jobs in your main Cloud Foundry deployment manifest (e.g. cf-manifest.yml):
+
+~~~
+jobs:
+#- instances: 4
+#  name: some_other_job
+#  ...
+- instances: 1            # add more instances if one job cannot keep up with the Firehose
+  name: datadog_nozzle_z1
+  networks:
+    - name: cf1           # some network you've configured elsewhere in the manifest
+  resource_pool: small_z1 # some resource_pool you've configured elsewhere in the manifest
+  templates:
+    - name: datadog-firehose-nozzle
+      release: datadog-firehose-nozzle
+  properties:
+    datadog:
+      api_key: <YOUR_DATADOG_API_KEY>
+      api_url: https://app.datadoghq.com/api/v1/series
+      flush_duration_seconds: 15          # seconds between flushes to Datadog. Default is 15.
+      metric_prefix: cloudfoundry.nozzle. # namespace for all nozzle metrics
+    loggregator:
+      # do NOT append '/firehose' or even a trailing slash to the URL; 'ws://<host>:<port>' will do
+      traffic_controller_url: <LOGGREGATOR_URL> # e.g. ws://traffic-controller.your-cf-domain.com:8081
+    nozzle:
+      deployment: <DEPLOYMENT_NAME>    # tags each firehose metric with 'deployment:<DEPLOYMENT_NAME>'
+      subscription_id: datadog-nozzle  # can be anything (firehose streams data evenly to all jobs using the same subscription_id)
+      # disable_access_control: true   # for development only
+      # insecure_ssl_skip_verify: true # for development only; enable if your UAA does not use a verifiable cert
+    uaa:
+      client: datadog-firehose-nozzle # client name you just configured
+      client_secret: <SECRET_YOU_JUST_CONFIGURED>
+      url: <UAA_URL> # e.g. https://uaa.your-cf-domain.com:8443
+~~~
+
+You can set `metrics_prefix` to anything you like, but if you want to use the default Cloud Foundry screenboard and to see metrics metadata in dashboards and graphs, set it to `cloudfoundry.nozzle.`.
```

Comment:
If we want to discourage it, we should not show the option in this snippet at all. Problem is, the default value is `cf.nozzle`, but we can change that in the nozzle release. What do you think @gmmeyer?